packages/coreutils/build.sh

```diff
@@ -3,10 +3,10 @@ TERMUX_PKG_DESCRIPTION="Basic file, shell and text manipulation utilities from t
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=9.5
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_REVISION=2
 TERMUX_PKG_SRCURL=https://mirrors.kernel.org/gnu/coreutils/coreutils-${TERMUX_PKG_VERSION}.tar.xz
 TERMUX_PKG_SHA256=cd328edeac92f6a665de9f323c93b712af1858bc2e0d88f3f7100469470a1b8a
-TERMUX_PKG_DEPENDS="libandroid-support, libgmp, libiconv"
+TERMUX_PKG_DEPENDS="libandroid-support, libgmp, libiconv, libandroid-selinux"
```

Comment:
Please sort the dependencies
```suggestion
TERMUX_PKG_DEPENDS="libandroid-selinux, libandroid-support, libgmp, libiconv"
```